IdentityServer/Startup.cs

```diff
@@ -99,6 +156,8 @@ public void ConfigureServices(IServiceCollection services)
             {
                 builder.AddDeveloperSigningCredential();
             }
+
+            // TODO tighten cors
```

Comment:
Is this something that needs to be done now (in this PR) or can this be done later?

Reply:
that is for later